client/src/main/scala/it/cwmp/client/ClientMain.scala

```diff
@@ -8,9 +8,21 @@ import it.cwmp.client.controller.ClientControllerActor
   * The client entry point
   */
 object ClientMain extends App {
+
+  // Terminal arguments check
+  require(args.length == 2,
+    """
+      Invalid number of arguments: I need
+      (1) the host address of the discovery service
+      (2) the port of the discovery service
+    """)
+
+  val discovery_host: String = args(0)
+  val discovery_port: Int = args(1) toInt
```

Comment:
Preferirei mantenere lo stile Java/Scala dei nomi e mettere in CamelCasing queste variabili

Reply:
Idem da tutte le altre parti dove le hai usate